### DISCUSSION TITLE
关于打开bat文件报错的问题

### DISCUSSION THREAD

**Discussion by neighbourfish** [BoomingTech/Piccolo#422] [Q&A] [answered]
<img width="720" alt="1" src="https://user-images.githubusercontent.com/47236362/211193610-1950cf01-0d37-40bb-b790-5068650aeedb.png">
使用VS2022运行也会报这样的错误，请问一下如何解决

**Comment by ShenMian** [comment, marked as answer]
该问题与 #390 相似.

**Reply by neighbourfish** [reply]
谢谢帮助~
